CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Guide: https://keepachangelog.com/en/1.0.0/
 
 <!-- Add changes for active work here -->
 
+- [Address] Change Country.ISO3166_1_alpha2 enum to public
```

Comment:
It looks like the change is not connected to the swiftlint. What do you think about putting this change into a separate pull request?

Reply:
I included this to fix the documentation in Discover+Options.swift ([link](https://github.com/mapbox/mapbox-search-ios/pull/239/files/feb31c3c86996fac04a9168ef6de279a11251641#diff-650b8f30b674b7bbc756814f2e046e711ab59dcdc8e852714a60e1a2b9afa007R20))